### PR TITLE
cmd: Fixup .clangd to use correct syntax

### PR DESCRIPTION
**Pull request by alexmurray**
Add is expected to be a list of options not a single element as per the
example in https://clangd.llvm.org/config.html#add

Signed-off-by: Alex Murray <alex.murray@canonical.com>

### CODE REVIEW

File: cmd/.clangd [hunk 1]
```diff
@@ -1,2 +1,2 @@
 CompileFlags:
-  Add: -I/usr/include/glib-2.0 -Wall -Wextra -Wmissing-prototypes -Wstrict-prototypes -Wno-missing-field-initializers -Wno-unused-parameter
\ No newline at end of file
+  Add: [-I/home/amurray/go/src/github.com/alexmurray/snapd-private/cmd, -I/usr/include/glib-2.0, -Wall, -Wextra, -Wmissing-prototypes, -Wstrict-prototypes, -Wno-missing-field-initializers, -Wno-unused-parameter]
```
Comment (bboozzoo) [inline cmd/.clangd:2]:
```suggestion
  Add: [-I/usr/include/glib-2.0, -Wall, -Wextra, -Wmissing-prototypes, -Wstrict-prototypes, -Wno-missing-field-initializers, -Wno-unused-parameter]
```

Comment (alexmurray) [reply cmd/.clangd:2]: ugh.. lol so embarrassed I didn't notice that before...

**Review by bboozzoo — APPROVED** [review]
Thanks!

**Review by anonymouse64 — APPROVED** [review]
lgtm

**Pull request merged**